vimwiki_markdown.py

```diff
@@ -93,12 +94,36 @@ def main():
     filename, _ = os.path.splitext(os.path.basename(INPUT_FILE))
     output_file = os.path.join(OUTPUT_DIR, filename + ".html")
 
-    extensions = ["fenced_code", "tables"]
-    extensions += os.getenv("VIMWIKI_MARKDOWN_EXTENSIONS", "").split(",")
-    extensions = set([e for e in extensions if e] + [CodeHiliteExtension()])
+    # parse extension into dict
+    try:
+        extensions_env = ast.literal_eval(
+            os.getenv(
+                "VIMWIKI_MARKDOWN_EXTENSIONS",
+                "{}",
+            )
+        )
+
+        # extract extension and extension_configs
+        if isinstance(extensions_env, list):
+            extensions = extensions_env
+            extension_configs = {}
```

Comment:
What if I'd like to have multiple extensions and set some options for each one (or few ones)?

Reply:
```vim
let $VIMWIKI_MARKDOWN_EXTENSIONS="{'codehilite': {'linenums': True}, 'fenced_code': {}, 'tables' : {}}"
```